### PR TITLE
[TASK] Replace @inject-Annotations with inject methods

### PR DESCRIPTION
**Pull request by pm-sschreiber**
Build is allowed to fail as tests still use old DatabaseConnection code (this will be resolved in a different PR).

### CODE REVIEW

File: Classes/Service/ExportService.php [hunk 1]
```diff
@@ -11,17 +11,31 @@
...
-     * @var PAGEmachine\Ats\Domain\Repository\JobRepository
-     * @inject
+     * @var JobRepository $jobRepository
      */
     protected $jobRepository;
 
+    /**
+     * @param ApplicationRepository $applicationRepository
+     */
+    public function injectApplicationRepository(ApplicationRepository $applicationRepository)
```
Comment (mbrodala) [inline Classes/Service/ExportService.php]: The import for `ApplicationRepository` is missing.

File: Classes/Service/ExportService.php [hunk 2]
```diff
@@ -11,17 +11,31 @@
...
+    {
+        $this->applicationRepository = $applicationRepository;
+    }
+
+    /**
+     * @param JobRepository $jobRepository
+     */
+    public function injectJobRepository(JobRepository $jobRepository)
```
Comment (mbrodala) [inline Classes/Service/ExportService.php]: The import for `JobRepository` is missing.

File: Classes/Service/ExportService.php (hunk 1 above)
Comment (pm-sschreiber) [reply Classes/Service/ExportService.php]: Added.

File: Classes/Service/ExportService.php (hunk 2 above)
Comment (pm-sschreiber) [reply Classes/Service/ExportService.php]: Added.

**Pull request merged**